server/tables/tables.go

```diff
@@ -97,6 +97,7 @@ type Invocation struct {
 	RepoURL      string `gorm:"index:repo_url_index"`
 	CommitSHA    string `gorm:"index:commit_sha_index"`
 	LastChunkId  string
+	BranchName   string `gorm:"index:branch_name_index"`
```

Comment:
I'm not sure we need this -- what query will hit this index?

Reply:
Those links just take me back to this CL -- but a good way to check would be to run "explain ...." on those queries in mysql and see if these are being used?

Reply:
yup, they get used when you visit the page for an individual repo, commit, or branch.